source/codegen/metadata/nidcpower/config.py

```diff
@@ -15,6 +15,209 @@
         'task': 'acquisition'
     },
     'custom_types': [
+        { 
+            'name': 'NIComplexNumber_struct',
+            'grpc_name': 'NIComplexNumber',
+            'fields': [
+                {
+                    'type': 'ViReal64',
+                    'name': 'real',
+                    'grpc_name': 'real'
+                },
+                {
+                    'type': 'ViReal64',
+                    'name': 'imaginary',
+                    'grpc_name': 'imaginary'
+                }
+            ]
+        },
+        {
+            'name': 'NILCRMeasurement_struct',
+            'grpc_name': 'NILCRMeasurement',
+            'fields': [
+                {
+                    'type': 'ViReal64',
+                    'name': 'Vdc',
```

Comment:
Is `name` field everywhere in fields supposed to follow camelCase? In that case, this should be vdc. Other names below should also be updated to `acVoltage`, `z`, `q`, `d`, `zMagnitude`, `ls`, `cs` and so on.

What about names of the custom types? Should they also be `niComplextNumber_struct`, `niLcrMeasurement` and so on?

Reply:
The 'name' tag is same as the field name in nidcpower.h, grpc names must be snake case as that is the convention followed for scope and switch grpc support. Even the names of custom types are same as what is being used in nidcpower.h